### PR TITLE
chore: cleanup notebook proxy feature branch

### PR DESCRIPTION
**Pull request by NicholasBlaskey**
## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: master/internal/rm/kubernetesrm/jobs.go [hunk 1]
```diff
@@ -308,7 +308,7 @@ func (j *jobsService) startClientSet() error {
 			j.tcpRouteInterfaces[ns] = alphaGatewayClientSet.TCPRoutes(ns)
 		}
 
-		// Don't think you can use the alphaGateway clientSet, because you can't.
```
Comment (hamidzr) [inline master/internal/rm/kubernetesrm/jobs.go:311]: :)

File: master/internal/config/resource_manager_config.go [hunk 2]
```diff
@@ -214,29 +214,30 @@ func (i *InternalTaskGatewayConfig) Validate() []error {
...
 	}
 
 	if err := check.IsValidK8sLabel(i.GatewayNamespace); err != nil {
-		errs = append(errs, errors.Wrap(err, "invalid gateway_namespace"))
+		errs = append(errs, fmt.Errorf("invalid gateway_namespace: %w", err))
 	}
 
-	if err := check.IsValidIP(i.GatewayIP); err != nil {
-		errs = append(errs, errors.Wrap(err, "invalid gateway_ip"))
+	// Don't validate the IP just check it is not empty so hostnames and the like can be used.
```
Comment (hamidzr) [inline master/internal/config/resource_manager_config.go:224]: we'd want to update the docs as well

File: tools/k8s/launch-minikube-with-gateway.sh [hunk 3]
```diff
@@ -5,20 +5,14 @@ set -e
...
     exit 1
 fi
 
-if [ ! -f $contour_config_file ]; then
-    echo "Gateway configuration file not found: $contour_config_file"
-    exit 1
-fi
-
 minikube start --profile $minikube_profile --kubernetes-version $K8S_VERSION
-kubectl apply -f $contour_config_file
```
[the remark this answers is not in the document]
Comment (hamidzr) [reply tools/k8s/launch-minikube-with-gateway.sh:21]: but having it lower for dev purposes is fine

**Review by hamidzr — APPROVED** [review]

File: tools/k8s/launch-minikube-with-gateway.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (NicholasBlaskey) [reply tools/k8s/launch-minikube-with-gateway.sh:21]: Yeah I couldn't tell the change easily. 

I think also a smaller limit is fine too.

File: master/internal/config/resource_manager_config.go (hunk 2 above)
Comment (NicholasBlaskey) [reply master/internal/config/resource_manager_config.go:224]: added a note that hostnames can be used

**Pull request merged**